Fw/Tlm/TlmPacket.hpp

```diff
@@ -23,19 +24,26 @@ namespace Fw {
             SerializeStatus serialize(SerializeBufferBase& buffer) const; //!< serialize contents
             // Buffer containing value must be remainder of buffer
             SerializeStatus deserialize(SerializeBufferBase& buffer);
-            // setters
-            void setId(FwChanIdType id);
-            void setTlmBuffer(TlmBuffer& buffer);
-            void setTimeTag(Time& timeTag);
-            // getters
-            FwChanIdType getId();
-            Time& getTimeTag();
-            TlmBuffer& getTlmBuffer();
-
-        PROTECTED:
-            FwChanIdType m_id; // !< Channel id
-            Fw::Time m_timeTag; // !< time tag
-            TlmBuffer m_tlmBuffer; // !< serialized data
+            // add telemetry value
```

Comment:
Should we have doxygen style comments, i.e., //! ?

Reply:
Added...

Reply:
Looks good! For some reason these changes don't appear in the GitHub PR view, but I see them in the repo.